src/adapters/catalyst-client.ts

```diff
@@ -27,6 +27,26 @@ export async function createCatalystClient({
     return createLambdasClient({ fetcher, url: lambdasServerUrl ?? loadBalancer })
   }
 
+  function getContentClientOrDefault(contentServerUrl?: string): ContentClient {
+    return createContentClient({ fetcher, url: contentServerUrl ?? loadBalancer })
+  }
+
+  function rotateContentServerClient<T>(
+    executeClientRequest: (client: ContentClient) => Promise<T>,
+    contentServerUrl?: string
+  ) {
+    const contentServers = shuffleArray(getCatalystServersFromCache(contractNetwork)).map((server) => server.address)
```

Comment:
Should we remove the `contentServerUrl` passed as parameter from the `contentServers`?